torch/_C/_dynamo/eval_frame.pyi

```diff
@@ -1,7 +1,10 @@
 import types
+from typing import NewType
 
 from torch._dynamo.types import DynamoCallback, DynamoGuardHook
 
+_PyInterpreterFrame = NewType("_PyInterpreterFrame", types.FrameType)
```

Comment:
from what I gather, we implement our own `FrameType`-like type for Python >= 3.11. So it's not actually an alias of FrameType, but still exposes the same interface.

(I will add this as a code comment)